aries-site/src/examples/components/card/CardExample.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box, Card, CardBody, CardFooter, Chart, Text, Stack } from 'grommet';
+import { Wifi } from 'grommet-icons';
+
+const mockData = Array(30)
+  .fill()
+  .map((_, index) => {
+    return {
+      value: [index, Math.random() * 100],
+    };
+  });
+
+const capacityWarnings = mockData.filter(datum => datum.value[1] >= 50).length;
+
+const gradient = [
+  { value: 0, color: 'status-ok' },
+  { value: 33, color: 'status-ok' },
+  { value: 67, color: 'status-warning' },
+  { value: 85, color: 'status-critical' },
+];
+
+const StyledCard = styled(Card)`
+  transition: all 0.3s ease-in-out;
+  :focus,
+  :hover {
+    transform: scale(1.01, 1.01);
+  }
+`;
+
+export const CardExample = () => {
+  const [isFocused, setIsFocused] = React.useState(false);
+
+  return (
+    <StyledCard
+      elevation={isFocused ? 'medium' : 'small'}
+      onClick={() => {
+        // eslint-disable-next-line no-alert
+        alert(
+          `Typically a click would route to a view with 
+            greater detail behind this summary information.`,
+        );
```

Comment:
This should help the alert message rendering:
```suggestion
        alert(`
        Typically a click would route to a view with 
        greater detail behind this summary information.`,
        `);
```